src/app.ts

```diff
@@ -773,6 +772,11 @@ export class App {
       actualVersion ? null : api.version
     );
 
+    fs.writeFileSync(
+      join(tmpDirPath, `${basename(destinationDirectory)}.json`),
+      JSON.stringify(api)
+    );
```

Comment:
Thoughts on adding a flag to `cli.ts` that would optionally write this source JSON directly in the relevant `types/<service>` directory? We could then enable it in the CI linting context. It seems less than ideal to have to manage a new `.tmp` directory.

There's a blessed way to [upload build artifacts](https://help.github.com/en/actions/configuring-and-managing-workflows/persisting-workflow-data-using-artifacts#uploading-build-and-test-artifacts). Coupled with my thought above, it might be cool to just upload the entire `types` directory, which we could download to do the same debugging.

Reply:
Good idea, let's continue conversation in #98